templates/community/debian.html

```diff
@@ -0,0 +1,27 @@
+{% extends "community/_base_community.html" %}
+
+{% block title %}Debian{% endblock %}
+{% block meta_description %}Debian is the rock on which Ubuntu is built.{% endblock meta_description %}
+
+{% block content %}
+
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Debian is the rock on which Ubuntu is built</h1>
+      <p>Ubuntu builds on the Debian architecture and infrastructure and collaborates widely with Debian developers, but there are important differences. Ubuntu has a distinctive user interface, a separate developer community (though many developers participate in both projects) and a different release process.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h3>About Debian</3>
```

Comment:
h3, not 3

Reply:
![image](https://user-images.githubusercontent.com/441217/36307729-554cb19a-1315-11e8-8378-248ebdb8a426.png)